### DISCUSSION TITLE
Slingbox Pro 500 - Plesae Help - Did not log into to get PW

### DISCUSSION THREAD

**Discussion by socelillyhammer** [GerryDazoo/Slinger#366] [Q&A] [answered]
Hi all and @GerryDazoo 

I need help with my slingbox 500s.  Unfortunately I found this group a little too late and didnt login to newatch intime to pull my PW and box data.

If anyone can help me to get my 500s up and running again, I'm willing to pay $1k (500 per each box - even if the solution is a copy paste method for the 2nd slingbox).

Please help.

**Comment by socelillyhammer** [comment]
Also to add I only had slingplayer web installed on my windows devices.

How f****ed am I?  or is there hope?

If you dont want pmt and just care to help out, always appreciated.

**Reply by rbf121** [reply]
I only had Slingplayer Desktop installed but I found some files here with passwords in the logs:
"%appdata%\Sling Media\Slingplayer Desktop\Logs\console.log"
"%appdata%\Sling Media\Slingplayer Desktop\cefCache\Local Storage\http_apollo.slingbox.com_0.localstorage"

If so, I was working on some utility scripts to extract passwords from those files.  (https://github.com/rbf121/Slinger-Utility)

Do you see anything in this folder?  C:\Users\UserName\AppData\Roaming\Sling Media

**Reply by socelillyhammer** [reply]
This folder C:\Users\UserName\AppData\Roaming\Sling Media
I found: alphonso.cfg

Opened it retreived adId and device ID but only for 1 of the 2 (nevertheless still better than nothing)

The other files and folders I dont have

**Reply by Dang638** [reply]
I am in the same boat web and android,ios user never installed windows player. Looked through all files do not see password.  Did the 500 also receive the kill firmware?

**Reply by socelillyhammer** [reply]
According to @GerryDazoo  500 did not receive the kill firmware. Did you try password to be "admin" ?  I still dont know how to figure out the setup.  But Im wondering if there is an electric technician engineer who can wire the hardware and get all the info.  That I'd pay for too. 

It seems all we need is the ID and the Passcode.

@GerryDazoo what do you rec if we have the ID.  How should we handle the PW?  Would  a hard reset work?

**Reply by sp222222** [reply]
do NOT HARD reset a 500!!!

**Reply by socelillyhammer** [reply]
@sp222222 understood. So where do we find our password if we did not log in on time?

**Reply by rbf121** [reply]
Just to confirm, the only Slingplayer client you ever used was the web slingplayer?  Like launch within internet explorer?

No other clients like iphone, firestick, etc?

**Reply by socelillyhammer** [reply]
I used it on firestick @rbf121

**Reply by socelillyhammer** [reply]
@rbf121 are you suggesting the password might be in the log file in the firestick?  If so I found this that I can search the firestick via ES explorer

https://www.firesticktricks.com/check-kodi-log-for-more-information.html

Found and checked an old android phone that had the sb apk.  the log file does not have any info.

**Reply by rbf121** [reply]
Right, the information might cached in there somewhere since the app loads the last used device.  But it's a matter of where and if you can even access it.  Maybe a long shot.

**Reply by Toei79** [reply]
you need check console.log ?

i able to find my passwords but its not working idk what its wrong.
last version can uncrypt passwords but still not luck, while i have one with reset password admin and its working.

**Reply by socelillyhammer** [reply]
@Dang638 @Toei79 @rbf121 @sp222222 Check out my new discussion.  I provided steps on how to get your pw to the 500!  I was shocked but it worked!  it worked!

**Reply by socelillyhammer** [reply]
@Toei79 see my other post discussion.  I explained how to find your password for the 500.  Its not lost.  The password is the encrypted one @GerryDazoo solved that problem for us with he updated program.  once you find your password (its a very long string) you alter the config file and for the password you enter "E1: followed by the long string"

Then check to see if you can login with VLC.

If you can, and that works, you can alter the video, frame rate, etc. within the config file and that should do the trick.

**Comment by socelillyhammer** [comment]


**Reply by Dang638** [reply]
so if the 500 did not get the kill firmware. Does factory resetting the 500 restore the password thats needed to admin?

**Reply by socelillyhammer** [reply]
I beleive so.  Let me screw around with it.

The finderId I found in the SPARCS_CACHE

I changed the config.ini file to the finderID that i picked up in the cache and the password I used was admin.

I ran the config file, it found my slingbox.  Now Im just waiting for the first stream.  So i have to figure out the video part to this whole thing and then after the video going figure out how to setup the server, vpn, and all that stuff

**Reply by sp222222** [reply]
You guys have no patience &  no tolarance for reading around before pulling stunts.  
Congrats, you now can add 30$ to get a remote to sit in front of the slingbox 500 AFTER you get slinGER setup and streaming..... no one's getting paid to look out for these disscsions threads. Sit back and be patience next time.

**Reply by socelillyhammer** [reply]
@sp222222 no one did a reset yet.  right now trying to figure out the password because we didnt login on time to get the info we needed before SB screwed us all.  $30 for the remote.  no problem.  I'll venmo you.  I'll give you $500 also to get me up and running.  Serious offer.

**Reply by Dang638** [reply]
> You guys have no patience & no tolarance for reading around before pulling stunts. Congrats, you now can add 30$ to get a remote to sit in front of the slingbox 500 AFTER you get slinGER setup and streaming..... no one's getting paid to look out for these disscsions threads. Sit back and be patience next time.

If this was directed towards me I simply asked the question. I have been reading here for the last 10 days if I missed something please inform me. I do not have a password and the only success I have seen was people who do or have a log file. Nether of which I have. I will wait i am in no hurry.

**Reply by socelillyhammer** [reply]
@Dang638 here is a step-by-step guide i found.  The problem is we need the password.  using admin does not work.
Once you have the password download VLC media player and follow the step-by-step guide.  Then you sould be all good to watch.

But without that PW we are fu**ed it seems

[Step-by-step.pdf](https://github.com/GerryDazoo/Slinger/files/10109404/Step-by-step.pdf)

**Reply by socelillyhammer** [reply]
@Dang638 This is the error I see once I try using VLC to connect.  We need that PW

![Screenshot 2022-11-28 221432](https://user-images.githubusercontent.com/119097385/204429968-34f24929-960e-4f69-84df-b48a102d60d8.png)

**Reply by GerryDazoo** [reply]
Dang, the answer is yes. It resets the password to "admin". BAd news is we don't yet know how to configure the videosource after the reset so your remote won't work

**Comment by Toei79** [comment]
same scenario , 
wondering what you access with slingbox anyway? i mean at very least you can access to your local tv with other stuff , 
im using tvheadend server, then i use some asus router that let me do local vpn when i away from home so i dont need use port forwarding, so i can see my ota channels.  sorry for my grammar. 

you will need this one if its directv . yep.

**Reply by socelillyhammer** [reply]
Always used the slingplayer web for the 500 and old cell phone with the apk on android but those have been discarded. Usually access the SB when traveling - im usually out of the country 4 months out of the year and it was a great item to use - news, sports, ocassional movie/show.  But mainly local news and sports.

Just checked out what tvheadend is and saw a video on YT.  All of this is too complicated for me as its not even close to what I do as a profession so its overlly complicated.

SB was simple and easy so Im trying to continue with it.

Once I can get my PWs if its possible, my next issue is the vpn, routers, port forwarding stuff etc. to watch remotely a whole other issue which ill be happy to pay to have someone set it all up.

**Reply by Toei79** [reply]
i see, yea you know this been forever,  i never know about linux always see all stuff like aliens,
then some catch my attention play youtube live channels on tivimate kinda like a directv on android.  
it can be hard yea when you dont know.

now i have hdhomerun, am fm radio (usb radio that you convert to stream link) (this one can be useful for play local radios remotely still need figure out to play remotely) , youtube, ustvgo in tivimate app for android, plus those free iptv pluto. 

i hope this get better here , so far its a lot made. imo

**Reply by socelillyhammer** [reply]
@Toei79 see my new discussion.  it worked!!

**Comment by Dang638** [comment]
![sli](https://user-images.githubusercontent.com/119362919/204654807-11b1c555-5e04-4433-a59c-50c31d2964ce.png)
 so this is where I am now I think I got the correct password but vlc media does not find it says unable to open the mrl

**Reply by socelillyhammer** [reply]
@Dang638 make sure you followed these steps to get your passwords if you couldnt log in to sb at the time to get your original info.
see the pdf if you need to use the desktop app to retreive your pw and id

[slingbox.pdf](https://github.com/GerryDazoo/Slinger/files/10117841/slingbox.pdf)

Once you retreived the password and finder ID you need to change the config.ini file.
your password in the config.ini file should include the "E1: and then the long string"
the finder id is your finder Id

Then save and exit

Then open the RunSling.bat file with notepad
where it says "cd slingbox" you need to change that directory.  For now I have my stuff in my downloads folder so my "cd slingbox" now reads 

cd C:\Users\(my username)\Downloads\Slinger-main\Slinger-main
slingbox_server.exe

Then save and exit.

Then once you have that up open and run the server app.

Then open your vlc in the file menu --> media --> netowrk stream and input the IP. 

If you have an issue there resend the server image again once you see you are connected.

**Reply by GerryDazoo** [reply]
Dang. Please post what you typed into VLC. The server logs don't show that it received any connection requests.

**Reply by Dang638** [reply]
> Dang. Please post what you typed into VLC. The server logs don't show that it received any connection requests.

Gerry and guys good job with this I got it running I had to use a different port as 8080 was taken or blocked by windows. Now to figure out the remote control. Thanks again

**Reply by socelillyhammer** [reply]
@Dang638 for your remote, just change your config file from
;include=remote.txt
to
include=remote.txt

or if your remote is in the Customs remots folder than have it read

include=CustomRemotes/ ______the.txt file name like FIOS_remote.txt

Then you can connect to http://[your ip address you have] : [your port]/Remote
so something like http://192.168.1.230:6543/remote

do that in the url of a webpage while your tv is streaming and it should bring up the remote.

let me know if the channels change and works as Im havin an issue with mine with verizon fios ipc4100 box on the fios one.

**Comment by ldbrouwer** [comment]
All, 
I'm trying to get my M2 up and running and had the same error msg as above (cmd: 0x67 err:0x2 8). I have the password that I used to log in to Slingbox - and I know it is right as I logged in on Nov 8). I do not have and E1:XXX string. In trying to follow the instructions from the "slingbox.pdf) above I tried to log in using my app on my iPad. It tells me an error has occurred on my Slingbox and to perform a hard reset. I haven't done that yet - as I don't want to mess it up. I'm not sure how I can retrieve my admin password using the PDF file - as I don't know where my console log file is... Any thoughts on how to best proceed?

**Reply by socelillyhammer** [reply]
Do you have a windows computer or mac that you have the desktop on? If not Id like to test a theory of mine but I would need you login and password to your slingbox

**Comment by ldbrouwer** [comment, marked as answer]
I have the slingbox server on a Raspberry Pi 4. I can provide the user id and password

**Reply by ldbrouwer** [reply]
sent

**Reply by socelillyhammer** [reply]
K. Give me 2 min.

**Reply by socelillyhammer** [reply]
is your finder id 2e921d3dab3e947d5e9e8abc00731634 ?

**Reply by socelillyhammer** [reply]
here is your E1 password

E1:C80EDB85ED4DB29BA9E5D8C12779E3E9A163A4AEB30E41F576EF0E96FFB936EC

**Reply by socelillyhammer** [reply]
@GerryDazoo @rbf121  @sp222222  Check this out.  You can find the E1 passwrod and unencrypted on still using my method.  I just tested it out for this guy on my computer.

**Reply by ldbrouwer** [reply]
S-Hammer - yes, the finder ID is correct. Thank you!

**Reply by socelillyhammer** [reply]
@ldbrouwer your regular unencrypted password is 199f0q305m4qj45

**Reply by socelillyhammer** [reply]
Can someone now please help me out with my verizon remote control issue for the fios one box?

**Reply by socelillyhammer** [reply]
check out posts #374 and #381
